### PR TITLE
Khe Sanh Update

### PR DESCRIPTION
**Pull request by Huckleberry87**
Updated Khe Sanh to new standard.

Overhaul; way too many updates to list by the numbers.

Overview:
Obj 1 - Reduce overall assets.
Obj 2 - Update vehicles to spawn system.
Obj 3 - Move everyone closer together.

-Spike Teams to the north of the runway. (From west to east, Tigers, Spike Team, UDT, ANZAC)
-Air Force on the west end of the runway. (633rd and Satan's Angels)
-ACAV is west of Mike Force, and is surrounded by Marines to the south, Blackhorse to the east, and ARVN to the west. 
-7th, Muskets, MACV, and MPs is East of Mike Force. 
-Mike Force is now the place to go to load onto helos. 
-Green Hornets is where you spawn to get a helo.

Added Dock teleporter to options, UDT is up at Khe Sanh, with the dock down at the river side.

CamLaoNam -
Moved TF to former 31st MEU base and Spike Team to former Tiger Force Hangar. Base rework complete.
MEU Arsenal and Teleporter now work."
ARVN has a statics spawner.
633rd Helos are now combined into a single spawner type across the 3 pads.
Added Teleporter and destination to CamLaoNam, for continuity.

KNOWN ISSUE: Carrier Spawn from respawn is ... not working as intended on Khe Sanh

### CODE REVIEW

**Review by Huckleberry87 — COMMENTED** [review]
Tested all locally. LGTM.

**Comment by dijksterhuis** [comment]
CamLaoNam updates look good. 2x things on Tigers

1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well
2. Airborne flags at the base? are they not meant to have the sog flag?

**Comment by dijksterhuis** [comment]
MPs mentioned wanting some transport trucks or the MP dirt rangers too? saw something about that somewhere

**Comment by dijksterhuis** [comment]
### khe sanh

switch these around so welcome billboard is the first one people see



flip UDT sign around so it is facing the runway (put it in near the base) --> match the other spike teams


blackhorse have two signs? leave the one facing the runway, get rid of the other?


non functional ARVN vehicle spawner


feels like there's a LOT of vehicles here. we might be able to get rid of some trucks?


no sign at USMC


monties map marker


7th cav map marker not captialised (should be a string table lookup??)


DC jet spawners not working


DC ground vehicles explosions -- probably need to move the apart a bit, find some flatter terrain or raise the spawn arrow up a bit



giving DC an airfield has made the map feel, dunno, like there's less up in that area. dunno how i feel about this. i quite like the last AO is the DC airfield (that doesn't have any planes, but there we are)

also -- there hasn't been DC Fixed Wing CAS on Khe Sanh before, and i'm wondering if we should keep it as the "no DC fixed wing cas" map. there's hella fewer trees on Khe Sanh, so it's prime Strella adventures.... don't really ever need CAS on that map as DC.

**Review by dijksterhuis — COMMENTED** [review]
added comments -- Tigers base edit and MPs truck or dirt rangers or whatever they asked for and i'm happy to merge and put on MF2 (we'll hold back with Khe Sanh as discussed)

**Comment by Huckleberry87** [comment]
> 1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well
> 2. Airborne flags at the base? are they not meant to have the sog flag?

1. Damn... ok.

2. Tigers are a part of the 101st. 1 BN / 327 BDE / 101 DIV

**Comment by dijksterhuis** [comment]
> 2. Tigers are a part of the 101st. 1 BN / 327 BDE / 101 DIV

errr, yes. erm. errr.

look! over there! in the opposite direction!

*runs away*

**Comment by dijksterhuis** [comment]
> > 1. Terrain hides for helos -- merge the 2x terrain hides for each helo spawn into one larger one -- remove the sandbag revetments as well

Didn't really explain this -- gonna be hard for some of them to land on those points and might as well combine the 2x into 1x larger one.

**Review by dijksterhuis — COMMENTED** [review]
2x more for khe sanh

boat dock arsenal / trash can not working


rust is gonna want some UDT whitelisted boats.... I just know it 😂 

we can maybe fit in another 2x boats i think -- so could do 3x UDT and 3x public similar to cam lao

**Comment by dijksterhuis** [comment]
comments addressed minus DC jets addition + motor pool

TODO -- trashcans!

**Comment by dijksterhuis** [comment]
simple object picnic tables etc added -- cam lao and khe sanh

**Comment by dijksterhuis** [comment]
Reminder -- libations issue. crates are falling through the map on cam lao nam

boxes should be raised up

khe sanh + CLN done

**Comment by dijksterhuis** [comment]
memorials added khe sanh

**Comment by dijksterhuis** [comment]
camlaonam hummingbird moved

**Comment by dijksterhuis** [comment]
khe sahn zones added and moved

**Comment by dijksterhuis** [comment]
mp troop transport vehicles

**Comment by dijksterhuis** [comment]
welcome sign added to CLN + KS

File: mission/config/subconfigs/vehicle_respawn_info.hpp [hunk 1]
```diff
@@ -968,6 +1011,14 @@ class spawn_point_types {
 					"vn_b_wheeled_m151_02_mp",
 				};
 			};
+			class transport {
```
Comment (dijksterhuis) [inline mission/config/subconfigs/vehicle_respawn_info.hpp]: duplicate name!

Comment (dijksterhuis) [reply mission/config/subconfigs/vehicle_respawn_info.hpp]: fixed with push

Comment (Huckleberry87) [reply mission/config/subconfigs/vehicle_respawn_info.hpp]: Damn! Good catch. I blame Renegade.

**Comment by dijksterhuis** [comment]
Also -- one broken 633rd spawn 

brkPain

**Comment by dijksterhuis** [comment]
> Also -- one broken 633rd spawn
> 
> brkPain

fixed

**Comment by dijksterhuis** [comment]
LGTM!

**Pull request merged**